src/Components/Comments.jsx

```diff
@@ -0,0 +1,16 @@
+function Comments({comment}) {
+
+    return (
+       <li className="comment-list">
+        <br></br>
+        <div className="comment">
+        <h3>{comment['author']}</h3>
+        <p>{comment['body']}</p>
+        <p>{comment['votes'] >= 0 ? `${comment['votes']} 👍` : `${comment['votes']} 👎`}</p>
+        <br></br>
+        </div>
+       </li>
+    )
```

Comment:
I like the conditional rendering of the 👍  /  👎 with the votes! 

I found the component name `Comments` confusing because it seems to be a single comment but 'Comments' implies it is the whole list 🤔